### DISCUSSION TITLE
Remaining problems with Open-ZFS update to 2.2.3 rc6

### DISCUSSION THREAD

**Discussion by guenther-alka** [openzfsonwindows/openzfs#388] [General]
I run an update from rc5 ..f74
Went through without problem.
Run installer a second time, also no problem

zfs get-all:  ok
echo "key"  | zfs load-key -L prompt tank/data  ok
(didn't work last time)

I have still problems
mount tank -> p:\
create a snap "@newest" for tank
open p:\.zfs\snapshot  ok
open p:\.zfs\snapshot\newest  explorer hangs like in rc.5

Remote replication via netcat between Windows and others
(Free-BSD, OSX, Proxmox, OmniOS), not sure if this is related to zfs or netcat

(tried nc64 1.12 as this is a simple free single binary, nmap netcat and cygwin64 netcat
-> invalid stream, bad magic number with partly success with nmap netcat

Question:
As zvol is disabled per default, how to enable if needed
- enable in Registry
-reboot?

- does this initiate a BSOD or only when updater is executed again?

**Comment by lundman** [comment]
Focus was the Aomei crashes, but sounds like that is better and we can look at other bugs again.
If you want to force load with zvol support, change Registry and reboot. It will probably BSOD 3/4 times at boot.
